src/wayland/shortcuts_inhibit/proto.hpp

```diff
@@ -37,21 +39,24 @@ class ShortcutsInhibitor
 public:
 	explicit ShortcutsInhibitor(::zwp_keyboard_shortcuts_inhibitor_v1* inhibitor, wl_surface* surface)
 	    : QtWayland::zwp_keyboard_shortcuts_inhibitor_v1(inhibitor)
-	    , mSurface(surface) {}
+	    , mSurface(surface)
+	    , bActive(false) {}
 
 	~ShortcutsInhibitor() override;
 	Q_DISABLE_COPY_MOVE(ShortcutsInhibitor);
 
-	[[nodiscard]] QBindable<bool> bindableActive() { return &this->bActive; }
+	[[nodiscard]] QBindable<bool> bindableActive() const { return &this->bActive; }
 	[[nodiscard]] bool isActive() const { return this->bActive; }
 	[[nodiscard]] wl_surface* surface() const { return this->mSurface; }
 
 signals:
 	void activeChanged();
 
 protected:
+	// NOLINTBEGIN(readability-identifier-naming)
 	void zwp_keyboard_shortcuts_inhibitor_v1_active() override;
 	void zwp_keyboard_shortcuts_inhibitor_v1_inactive() override;
+	// NOLINTEND(readability-identifier-naming)
```

Comment:
This lint shouldnt trigger in the first place for overrides

Reply:
All that lint does is thisptr usage. Depending on your version of clang-tidy vs ci's you might get different reports.